scenarios/quotation-flow.golden

```diff
@@ -18,6 +18,7 @@ Modifying default user.
 11: forms quotation new --client charlie
 Quotation form created: TBPJLIUG
 13: forms quotation submit TBPJLIUG
+# TODO Add either a --dry-form option or a "validate" subcommand to execute only the validation code
```

Comment:
The command `forms quotation validate ` actually already exists. It is now called in this PR https://github.com/hypered/curiosity/pull/123/files.